demisto_sdk/commands/common/tests/tools_test.py

```diff
@@ -2851,3 +2853,194 @@ def test_sha1_dir():
+                {
+                    "integrations": "TestCreateDuplicates",
+                    "playbookID": "TestCreateDuplicates Test",
+                },
+            ],
+        ),
+        (
+            "Account Enrichment",
+            "integration",
+            ["No tests"],
+            False,
+            [
+                {"integrations": ["PagerDuty v1"], "playbookID": "PagerDutyV1 Test"},
+                {
+                    "integrations": ["PagerDuty v1", "PagerDuty v2"],
+                    "playbookID": "PagerDuty Test",
+                },
+                {"integrations": "PagerDuty v2", "playbookID": "PagerDutyV2 Test"},
+                {"integrations": "PagerDuty v2", "playbookID": "PagerDutyV2 Test"},
+                {
+                    "integrations": "TestCreateDuplicates",
+                    "playbookID": "TestCreateDuplicates Test",
+                },
+            ],
+        ),
+        (
+            "Account Enrichment Test",
+            "playbook",
+            ["No tests"],
+            False,
+            [
+                {"integrations": ["PagerDuty v1"], "playbookID": "PagerDutyV1 Test"},
+                {
+                    "integrations": ["PagerDuty v1", "PagerDuty v2"],
+                    "playbookID": "PagerDuty Test",
+                },
+                {"integrations": "PagerDuty v2", "playbookID": "PagerDutyV2 Test"},
+                {"integrations": "PagerDuty v2", "playbookID": "PagerDutyV2 Test"},
+                {
+                    "integrations": "TestCreateDuplicates",
+                    "playbookID": "TestCreateDuplicates Test",
+                },
+            ],
+        ),
+    ],
+)
+def test_search_and_delete_from_conf(
+    content_item_id,
+    file_type,
+    test_playbooks,
+    no_test_playbooks_explicitly,
+    expected_test_list,
+):
+
+    """
+    Given:
+          content_item_id, file_type, test_playbooks, no_test_playbooks_explicitly
+        - Case A: PagerDuty v2 integration without tests.
+        - Case B: PagerDuty v2 integration with tests.
+        - Case C: Account Enrichment integration without tests.
+        - Case D: Account Enrichment playbook without tests.
+
+    When:
+        - check that the test_search_and_delete_from_conf works as expected.
+
+    Then:
+        - Case A: Delete PagerDuty v2 integration.
+        - Case B: Delete PagerDuty v2 integration.
+        - Case C: Delete Account Enrichment integration.
+        - Case D: Delete Account Enrichment integration.
+    """
+    CONF_JSON_ORIGINAL_CONTENT = {
+        "tests": [
+            {"integrations": ["PagerDuty v1"], "playbookID": "PagerDutyV1 Test"},
+            {
+                "integrations": ["PagerDuty v1", "PagerDuty v2"],
+                "playbookID": "PagerDuty Test",
+            },
+            {"integrations": "PagerDuty v2", "playbookID": "PagerDutyV2 Test"},
+            {"integrations": "PagerDuty v2", "playbookID": "PagerDutyV2 Test"},
```

Comment:
Duplicate?

Reply:
Yes, it is on purpose, in order to check adage cases.